SoftLayer/CLI/licenses/create.py

```diff
@@ -0,0 +1,33 @@
+"""Order/create a vwmare licenses."""
+# :licenses: MIT, see LICENSE for more details.
+
+import click
+from SoftLayer.managers import ordering
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command()
+@click.option('--key', '-k', required=True, prompt=True, help="The License Key for this specific Account License.")
+@click.option('--datacenter', '-d', required=True, prompt=True, help="Datacenter shortname")
+@environment.pass_env
+def cli(env, key, datacenter):
+    """Order/create a vlan instance."""
+
+    complex_type = 'SoftLayer_Container_Product_Order_Software_License'
+    item_package = [key]
+
+    ordering_manager = ordering.OrderingManager(env.client)
+    result = ordering_manager.place_order(package_keyname='SOFTWARE_LICENSE_PACKAGE',
```

Comment:
It would be better to move this `place_order `request to the `LicenseManager `to a specific method and set the data required for the order, such as `complex_type`, etc.